control-plane/tests/test_cli_inspection_workflow_family.py

```
@@ -0,0 +1,1258 @@
+        )
+        self.assertEqual(
+            payload["records"][0]["substrate_detection_record_ids"],
+            ["wazuh:1731594986.4931506"],
+        )
+
+    def test_cli_renders_reviewed_wazuh_alert_detail_view(self) -> None:
+        store, _ = make_store()
+        service = AegisOpsControlPlaneService(
+            RuntimeConfig(
+                postgres_dsn="postgresql://control-plane.local/aegisops",
+                wazuh_ingest_shared_secret="reviewed-shared-secret",
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to argument: "wazuh_ingest_shared_secret"

(S106)